flink_job.go

```diff
@@ -0,0 +1,131 @@
+package aiven
+
+type (
+	// FlinkTableHandler aiven go-client handler for Flink Jobs
+	FlinkJobHandler struct {
+		client *Client
+	}
+
+	// CreateFlinkJobRequest Aiven API requess
+	// POST https://api.aiven.io/v1/project/<project>/service/<service_name>/flink/job
+	CreateFlinkJobRequest struct {
+		JobName   string   `json:"job_name,omitempty"`
+		Statement string   `json:"statement"`
+		Tables    []string `json:"tables"`
+	}
+
+	// CreateFlinkJobResponse Aiven API response
+	// POST https://api.aiven.io/v1/project/<project>/service/<service_name>/flink/job
+	CreateFlinkJobResponse struct {
+		APIResponse
+
+		JobName string `json:"job_name"`
+		JobId   string `json:"job_id"`
+	}
+
+	// DeleteFlinkJobRequest Aiven API request
```

Comment:
Probably it should be a `PatchFlinkJobRequest` here?

Reply:
:heavy_check_mark: